docs/releases/overview.md

```diff
@@ -7,23 +7,86 @@ The Vega software is publicly available on [GitHub](https://github.com/vegaproto
 
 [**Vega core software**](#vega-core-software) - Below, find a summary of each version's features and breaking changes.
 
-See the full release notes on [GitHub](https://github.com/vegaprotocol/vega/releases).
+From 0.54.0 the core repository also holds the data node and CLI wallet code, therefore the following code is included in the releases:
 
-[**Data node APIs on GitHub**](https://github.com/vegaprotocol/data-node/releases) - The data node APIs allow for querying for historic information and for snapshots of the current state of the systems.
+- The data node APIs allow for querying for historic information and for snapshots of the current state of the systems.
+- The code for the Vega Wallet CLI app is open source, and you can read about the contents of each release on the Vega Wallet repo.
```

Comment:
Since the Wallet software has been OS for ages, do you think we need to call it out?